components/motor/fake/motor.go

```diff
@@ -18,7 +19,17 @@ import (
 	"go.viam.com/rdk/resource"
 )
 
-var model = resource.DefaultModelFamily.WithModel("fake")
+var (
+	motorModel    = resource.DefaultModelFamily.WithModel("fake")
+	b             board.Board
```

Comment:
:grimacing: Global mutable state is usually a bad idea. What happens if there are two fake motors on a single robot? Won't the second one mess this up for the first one?

This variable is only used within a single function, and is re-initialized before each use. I wonder if it can be made a local variable within there instead.

Reply:
sure!